src/@sdk/mutations/index.ts

```diff
@@ -100,17 +53,6 @@ export const MUTATIONS = {
       mutation: User.accountUpdate,
       ...options,
     }),
-  AddCheckoutPromoCode: <TCacheShape>(
```

Comment:
No need to update types?

Reply:
What types?

Reply:
Mutations types are still in use in `src/@sdk/network/index.ts`